apps/api/src/app/workflows-v1/usecases/update-workflow/update-workflow.usecase.ts

```diff
@@ -167,8 +168,8 @@ export class UpdateWorkflow {
         updatePayload.validatePayload = command.validatePayload;
       }
 
-      if (command.status) {
```

Comment:
i am not sure why we allowed to pass the status directly as it is coupled to the `active` state,  and to `steps` to locate the issues.